tests/test_search_amebsa.c

```
@@ -108,8 +111,8 @@ static int run_amebsa_once(real temptr, long seed, amebsa_result *out)
 static int test_reproducible_with_fixed_seed(void)
 {
     amebsa_result r1, r2;
```

Clang (14.0.6)

[warning] multiple declarations in a single statement reduces readability

(readability-isolate-declaration)